cmd/minikube/cmd/config/profile_list.go

```diff
@@ -179,11 +168,77 @@ func profilesToTableData(profiles []*config.Profile) [][]string {
 			k = "*"
 		}
 		if isDetailed {
-			data = append(data, []string{p.Name, p.Config.Driver, p.Config.KubernetesConfig.ContainerRuntime,
-				cpIP, strconv.Itoa(cpPort), k8sVersion, p.Status, strconv.Itoa(len(p.Config.Nodes)), c, k})
+			if p.Status == "OK" {
+				data = append(data, []string{
+					fmt.Sprintf("%s%s%s", constants.Enabled, p.Name, constants.Default),
```

Comment:
we should have a style func that does 
style.Normal( white color)
style.Enabled(
Style.Trouble(